packages/fetchai/skills/gym/rl_agent.py

```diff
@@ -159,7 +159,7 @@ def _get_random_next_good(self) -> int:
 
         :return: a random good
         """
-        return random.choice(list(self.good_price_models.keys()))
+        return random.choice(list(self.good_price_models.keys()))  # nosec
```

Comment:
Note: https://bandit.readthedocs.io/en/latest/blacklists/blacklist_calls.html#b311-random
Here, the usage is not security critical so we skip it.